debezium-core/src/main/java/io/debezium/relational/RelationalSnapshotChangeEventSource.java

```diff
@@ -153,6 +153,7 @@ public SnapshotResult<O> doExecute(ChangeEventSourceContext context, O previousO
 
             if (snapshottingTask.snapshotData()) {
                 LOGGER.info("Snapshot step 7 - Snapshotting data");
+                Thread.sleep(2000);
```

Comment:
Gasp, is there any way to not do that?

Reply:
@gunnarmorling Sorry for that, it is garbage from a testing of a different PR :-).
I going to remove it right away.
Still please read https://issues.redhat.com/browse/DBZ-6047?focusedId=22479922&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22479922

Reply:
@gunnarmorling Is it better now? :-)

Reply:
Yepp, much better, thanks :)